### PR TITLE
IA-2195 bump scalatest for all rawls projects

### PR DESCRIPTION
**Pull request by Qi77Qi**
In an effort trying to cross build rawls-model, which workbenchServiceTest relies on.
- Bump scala test version. Following [this](https://github.com/scalatest/autofix/tree/master/3.1.x#autofix-for-scalatest-31x)
- bump workbenchModel version to latest in wb-libs

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
1. bump scalatest version
  

* **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by kshakir — APPROVED** [review]

File: project/Dependencies.scala [hunk 1]
```diff
@@ -85,15 +85,15 @@ object Dependencies {
   val liquibaseCore: ModuleID =   "org.liquibase"                 % "liquibase-core"        % "3.5.3"
   val logbackClassic: ModuleID =  "ch.qos.logback"                % "logback-classic"       % "1.2.2"
   val scalaUri: ModuleID =        "io.lemonlabs"                  %% "scala-uri"            % "0.5.3"
-  val scalatest: ModuleID =       "org.scalatest"                 %% "scalatest"            % "3.0.1" % "test"
-  val mockito: ModuleID =         "org.mockito"                   % "mockito-core"          % "2.7.22" % "test"
+  val scalatest: ModuleID =       "org.scalatest"                 %% "scalatest"            % "3.2.2" % "test"
+  val mockito: ModuleID =         "org.scalatestplus" %% "mockito-3-3" % "3.2.2.0" % Test
```
Comment (davidangb) [inline project/Dependencies.scala]: nit: whitespace formatting

**Review by davidangb — APPROVED** [review]

**Comment by davidangb** [comment]
hmmm ... though obviously this needs to resolve the test failures!

**Comment by Qi77Qi** [comment]
@davidangb yea, haven't had time to look into tests yet...2 unit tests failed, both around mockito

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
jenkins failed due to ci is trying to publish rawls-models to scala 2.11, and we don't have `"workbench-model" % "0.14-65bba14"` for 2.11

I'm guessing 2.11 is for orchestration, which is moving away from scala 2.11. So I think it should be ok to update CI to not publish rawls-model for 2.11, what do you guys think @davidangb @kshakir

**Comment by davidangb** [comment]
@MatthewBemis what do you think of Qi's question above?

**Comment by kshakir** [comment]
I think in the past year I tripped over 2.11 being used someplace else in the Firecloud/Terra multi-repo. I can't recall where though... may have been something like [Agora still using the 2.11 based scala-baseimage](https://github.com/broadinstitute/agora/blob/develop/Dockerfile#L3-L4) in some weird way for either builds, tests, CI, or something... Sorry I can't recall more specifics.

Still, I say if Rawls is happy, go for it.

Just an FYI @rsasch @salonishah11 @cjllanwarne @aednichols

**Comment by davidangb** [comment]
I bet Thurloe is still 2.11, but I don't know if that matters

**Comment by MatthewBemis** [comment]
With the spray -> akka-http migration mostly done for orch I think we're not far off from being on scala 2.12+. I do get a little nervous about suddenly needing to adopt a new rawls model for orch before we get there, especially since the Cloud Integration team has recently been making changes to some of these models (they just bumped the rawls model in orch last week) so we could be in a tricky position if that happens. I think there should be ways to work around that if it does happen (even if it's by duplicating the models in orch until we finally push it to 2.12+), and I realize that our backs are already sort of up against that wall because workbench-model has dropped 2.11 support.

For thurloe, it's small enough where I think we could get to 2.12/akka-http quickly enough if we had to and I don't think we anticipate needing to change any of the rawls models that thurloe uses any time soon (I think it's mostly notification-related models).

@Qi77Qi I say go for it. In the unlikely event that we need to adopt new models in orch before we get to 2.12+ we can definitely work around it even if it's temporarily ugly.

**Comment by Qi77Qi** [comment]
thank you all for the comments/reviews...going to merge this soon...happy to help with upgrading orch to 2.12 whenever needed as well @MatthewBemis

**Pull request merged**